<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCarDialogClass</class>
 <widget class="QDialog" name="AddCarDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddCarDialog</string>
  </property>
  <widget class="QPlainTextEdit" name="productionYearEditText">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="placeholderText">
    <string notr="true">1999</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>140</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="plainText">
    <string>price_per_hour</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="pricePerDaySpinbox">
   <property name="geometry">
    <rect>
     <x>147</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>200.000000000000000</double>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="modelEditText">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="placeholderText">
    <string>model</string>
   </property>
  </widget>
  <widget class="QComboBox" name="transmissionCombobox">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>140</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="placeholderText">
    <string>transmission</string>
   </property>
   <item>
    <property name="text">
     <string>manual</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>automatic</string>
    </property>
   </item>
  </widget>
  <widget class="QDoubleSpinBox" name="capacitySpinbox">
   <property name="geometry">
    <rect>
     <x>457</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.050000000000000</double>
   </property>
   <property name="maximum">
    <double>15.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
   <property name="value">
    <double>0.900000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="pricePerHourSpinbox">
   <property name="geometry">
    <rect>
     <x>147</x>
     <y>140</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.100000000000000</double>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>15.000000000000000</double>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>412</x>
     <y>360</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_7">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>347</x>
     <y>60</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="plainText">
    <string>power</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="powerSpinbox">
   <property name="geometry">
    <rect>
     <x>457</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>2000</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
  </widget>
  <widget class="QComboBox" name="fuelTypeCombobox">
   <property name="geometry">
    <rect>
     <x>427</x>
     <y>180</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="placeholderText">
    <string>fuel_type</string>
   </property>
   <item>
    <property name="text">
     <string>petrol</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>petrol_and_lpg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>diesel</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>electric</string>
    </property>
   </item>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_5">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>190</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="plainText">
    <string>price_per_day</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_6">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>347</x>
     <y>10</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="plainText">
    <string>capacity</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="manufacturerEditText">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="placeholderText">
    <string>manufacturer</string>
   </property>
  </widget>
  <widget class="QComboBox" name="driveCombobox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>100</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="placeholderText">
    <string>drive</string>
   </property>
   <item>
    <property name="text">
     <string>front</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>rear</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>all</string>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
